vit-servicing-station-tests/src/tests/bootstrap/arguments/out_settings.rs

```diff
@@ -0,0 +1,77 @@
+use crate::common::{
+    paths::BLOCK0_BIN,
+    startup::{
+        empty_db,
+        server::{dump_settings, load_settings, BootstrapCommandBuilder, ServerSettingsBuilder},
+    },
+};
+use assert_cmd::assert::OutputAssertExt;
+use assert_fs::{fixture::PathChild, TempDir};
+use std::path::PathBuf;
+use vit_servicing_station_lib::server::settings::ServiceSettings;
+
+#[ignore]
```

Comment:
Is this one failing?

Reply:
I've reported "question" for that: https://github.com/input-output-hk/vit-servicing-station/issues/62

Reply:
I opened a fix in #63.

Reply:
@dkijania this shouldn't be ignored anymore right?